blackbox/docs/migrating.txt

```diff
@@ -0,0 +1,125 @@
+.. highlight:: psql
+
+=========
+Migrating
+=========
+
+Some versions of CrateDB require version-specific migration steps.
+
+Please see the section that corresponds to the version you are migrating to for
+instructions.
+
+Migrating to 1.1
+================
+
+CrateDB 1.1 introduces a new internal table format that is backwards
+incompatible with previous table formats. Future versions of CrateDB will only
+support this new table format. CrateDB 1.1 will be the only release that can
+read all older table formats *and* the new table format.
+
+CrateDB 1.1 includes new cluster checks that will tell you which tables need
+updating or recreating if you plan to use future releases of CrateDB. Once this
+has been done, your tables will be forward compatible with future releases.
+
+To get started, upgrade your cluster like normal. When CrateDB 1.1 is running,
+you may now see that your cluster checks are yellow rather than green in the
+status bar at the top of your `admin console`_. Select the status checks icon
+for more information.
+
+Depending on your situation, you may then be told that you should either
+upgrade or recreate some of your tables.
+
+.. figure:: _static/upgrade-cluster-checks.png
+   :align: center
+
+   *Figure 1*: Both cluster checks are failing in this example.
+
+If your cluster checks are green, there is nothing for you to do. Otherwise,
+continue to the section `Preparation`_.
+
+Preparation
```

Comment:
maybe we need to skip this step as the upgrade is considered safe and for the recreate you export your schema and data anyways. @seut right?